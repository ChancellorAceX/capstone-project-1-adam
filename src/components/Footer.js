import React from 'react';
import { Link } from 'react-router-dom';
import '../styles/Footer.css';
import magnifyingglass from '../resources/magnifying-glass-2.png';
import mapicon from '../resources/map-2.png';


export default function Footer(props) {
  return (
    <footer>
      <Link to='/bestiary'><img className='footer' src={magnifyingglass} alt='alt' /></Link>
      <Link to='/login'><img className='footer' src={mapicon} alt='alt' /></Link>
    </footer>
  );
}