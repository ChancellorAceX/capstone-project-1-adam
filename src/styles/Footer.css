* {
  box-sizing: border-box;
}

footer {
  position:fixed;
  bottom:0;
  right:0;
  margin:0;
  padding:0;
  height:8vh;
  width:100vw;
  border-top: 1px black solid;
  display:flex;
  justify-content: space-evenly;
  align-items: center;
  background-color: #333333;
}

img.footer {
  height:5vh;
  margin:1.5vh;
}