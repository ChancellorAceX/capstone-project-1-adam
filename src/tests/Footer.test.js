import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../components/Footer';

describe('Footer component', () => {
  it('renders the full footer component', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});