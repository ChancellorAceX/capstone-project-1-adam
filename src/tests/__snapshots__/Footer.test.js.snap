// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the full footer component 1`] = `
<footer>
  <Link
    to="/"
  >
    <img
      alt="alt"
      className="footer"
      src="magnifying-glass-2.png"
    />
  </Link>
  <Link
    to="/login"
  >
    <img
      alt="alt"
      className="footer"
      src="map-2.png"
    />
  </Link>
</footer>
`;
